app/server.go

```diff
@@ -747,33 +750,37 @@ func doSessionCleanup(s *Server) {
 	s.Store.Session().Cleanup(model.GetMillis(), SESSIONS_CLEANUP_BATCH_SIZE)
 }
 
-func (s *Server) StartElasticsearch() {
-	s.Go(func() {
-		if err := s.Elasticsearch.Start(); err != nil {
-			s.Log.Error(err.Error())
-		}
-	})
+func (s *Server) StartSearchEngine() {
```

Comment:
I can't see a `StopSearchEngine`. Shouldn't we call that method in the `Shutdown`? And if yes we should call `RemoveConfigListener` as well

Reply:
What about the `StopSearchEngine`?

Reply:
Added, I think is cleaner now.